dockstore-webservice/src/main/resources/migrations.1.6.0.xml

```diff
@@ -279,4 +279,16 @@
             <column name="topic" type="varchar(255 BYTE)"/>
         </addColumn>
     </changeSet>
+
+    <changeSet author="charlesoverbeck (generated)" id="1551393509564-1">
```

Comment:
change the id to something more readable (on the off-chance we need to debug migrations by looking at the DB)